src/main/java/com/box/sdk/BoxStoragePolicy.java

```diff
@@ -0,0 +1,162 @@
+package com.box.sdk;
+
+import java.net.URL;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+import com.eclipsesource.json.ParseException;
+
+/**
+ *  Represents a BoxStoragePolicy.
+ */
+@BoxResourceType("BoxStoragePolicy")
```

Comment:
I think the string in this annotation is supposed to be the Box type, e.g. `"storage_policy"`